web-marketplace/src/components/organisms/RegistryLayout/RegistryLayout.Header.tsx

```diff
@@ -47,6 +47,13 @@ import {
 // import { LanguageSwitcher } from './RegistryLayout.LanguageSwitcher';
 import { getAddress, getProfile } from './RegistryLayout.utils';
 
+const getProfileLink = (activeAccount: any, wallet: any): string => {
```

Comment:
can we avoid using `any`? here you can use `Maybe<AccountFieldsFragment>` (as in `getDisplayAccountOrAddress` function) and `Wallet`